colossalai/inference/dynamic_batching/stas.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+
+class Stats:
```

Comment:
The filename would be `stats.py`, right?

Reply:
for sure